### ISSUE TITLE
MPEG-DASH duration mismatch between AdaptationSets.

### ISSUE THREAD

**Issue by alex-rsk** [gpac/gpac#2966]
- [x] I looked for a similar issue and couldn't find any.
- [ ] I tried with the latest version of GPACф. Installers available at https://gpac.io/downloads/gpac-nightly-builds/
- [x] I give enough information for contributors to reproduce my issue (meaningful title, github labels, platform and compiler, command-line ...). I can share files anonymously with this dropbox: https://www.mediafire.com/filedrop/filedrop_hosted.php?drop=eec9e058a9486fe4e99c33021481d9e1826ca9dbc242a6cfaab0fe95da5e5d95

Given command for dashing:

```
MP4Box -dash 2560 -dash-ctx /dev/shm/broadcast/2EqBK8l1p6/context.txt -profile dashavc264.live 
--utcs="https://time.akamai.com/?iso&ms" -mpd-refresh 3 -dynamic -bs-switching no 
-segment-timeline -url-template -insert-utc 
-min-buffer 31  --sflush=end -time-shift 61.44  -tmp /tmp/2EqBK8l1p6 
2EqBK8l1p6.mp4#video:#Representation=norm 2EqBK8l1p6.mp4#audio 2EqBK8l1p6_en.vtt:id=3:#Role=subtitle:#Language=en:dur=10.24:#Bitrate=100 
-out /dev/shm/broadcast/2EqBK8l1p6/manifest.mpd  
```                                                                      
MP4Box throws warning that AdaptationSets are of unequal durations                                         
```
...
[Dasher] Adaptation sets in period are of unequal duration min 17.92 max 20.48 seconds 
[Dasher] updated period DID1 duration 20480 MPD time 20480
...
[Dasher] End of MPD (no more active streams)          
```
Each time difference is  2.56 sec - a one segment length

MP$box -info says that tracks are of equal duration:
```
# Track 1 Info - ID 1 - TimeScale 12800
Media Duration 00:00:10.280 
Track has 1 edits: track duration is 00:00:10.240

# Track 2 Info - ID 2 - TimeScale 8000
Media Duration 00:00:10.239 

```

Each vtt file is also aligned
```
WEBVTT

00:00:00.100  --> 00:00:05.120  align:center line:80% size:80%
<c.sp-1>Why was six afraid of seven?</c>

00:00:05.219  --> 00:00:10.240  align:center line:80% size:80%
<c.sp-2>Because seven eight nine!</c>
```

**Comment by alex-rsk** [comment]
Visualization of stream in Shaka player.  Some strange offset of text stream is clearly visible:

![image](https://github.com/user-attachments/assets/0988a5dd-5209-4d15-8a84-1412fa9d04da)

**Comment by alex-rsk** [comment]
@jeanlf  @rbouqueau need your help, please respond.

**Comment by jeanlf** [comment]
could you upload some sample files so that we can reproduce ?

**Comment by alex-rsk** [comment]
Sure. I uploaded segments, texts and log of dasher session.

https://www.dropbox.com/scl/fo/rsyuwlp3vubngveoslp3v/AB6gKCZZcBORB10ChsIGLks?rlkey=spyrs7a68ty29ic6ott6gbeht&st=jjfhbt1o&dl=0

**Comment by alex-rsk** [comment]
Solved, when changed :dur=10.24 to :duration=2.56.
